CONTRIBUTING.md

```diff
@@ -40,19 +40,35 @@ Please note we have a code of conduct, please follow it in all your interactions
 Pull Request Process
 
  
-    1.  Create your own branch with convention 
-        "name_task" convention. Do changes and commits from 
-        your end and create a pull request to add the branch.
+    1.  Create your own branch with convention "name_of_the_task@yourfirstname"
+
+            git checkout -b <you_branch_name>
+
+        Do changes and commit on that branch using:
+
+            git add <list_of_files_to_be_committed> or git add .
+            git commit -a -m "commit_message"
+            git push origin <your_branch_name>
+         
+        Once committed the changes and pushed, create a pull request with that branch. 
+        Please make sure to add #issue_number in the description of the PR.
+        Also make sure to provide the list of features added through the PR.
     
-    2.  Do verify the changes in branch are localised to 
-        issues  addressed in your own system. Before a brach push
-        or PR, do a pull from remote master to get recent changes
+    2.  Do verify that your branch have got all the changes that are done in the main repo.
+        Before a branch push or PR, do a pull from remote master to get recent changes by following steps:
+
+            git remote add upstream https://github.com/rajatkb/Conference-Notify.git
+            git remote -v
+            git fetch upstream
+            git checkout master
+            git merge upstream/master
+            git push origin master
```

Comment:
You don't need to push to the master branch. Split this process into two steps. Any one of the following can be used to pull the latest changes from master:
- Before creating branch pull latest changes into master using the following commands:
        1) git checkout master
        2) git pull
        3) git checkout -b <your-branch>

- Create your branch from the master branch, commit your changes and then rebase your branch from master:
        1) git checkout master
        2) git checkout -b <you_branch_name>
        3) Commit your changes:
             git add .
             git commit -a -m "commit_message"
        4) Rebase your branch from master to pull in the latest changes:
             git pull origin master --rebase
        5) Push your branch to origin

Reply:
@MahamArif If one follows the above steps, git remote add upstream https://github.com/rajatkb/Conference-Notify.git will be required or not? I think it should be required.

Reply:
@Kajol-Kumari It will be required as we are using the forking scheme here. I have updated my above comment. Please check.

Reply:
Why I asked here to rebase your task branch with master is, because between the duration of creating your branch, doing your changes and pushing back your branch to origin, The original repo may have been updated because some other PR can be merged. So it is better to always update your master branch and rebase your task branch before pushing your branch and making the pull request.

Reply:
okay @MahamArif. Thanks for the suggestion. I have made the changes, please have a look at it and if any further modification is required please let me know.